sql/core/src/main/scala/org/apache/spark/sql/execution/command/ddl.scala

```diff
@@ -703,4 +703,17 @@ object DDLUtils {
       }
     }
   }
+
+  /**
+   * Masking credentials in the option lists. For example, in the sql plan explain output
+   * for JDBC data sources.
+   */
+  def maskCredentials(options: Map[String, String]): Map[String, String] = {
+    options.map {
```

Comment:
we should consolidate this code path with the one above. Otherwise the two will diverge over time. Perhaps add them to some utils class?